### PR TITLE
Introduce webpack

### PR DESCRIPTION
**Pull request by korenkovv**

### CODE REVIEW

File: WebClient/doc/debugging.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# Debugging hints
+
+## Trigger functionality from developer console
+
+In Chrome or Explorer or any other browser one can execute javascript code from Developer Console and trigger application functionality.
+
+An important detail is that classes can't be accessible directly from the global scope because our code is wrapped in modules. To access a class or any other object or function exported by some module you can use `require(modulePath)` function. This function is available in the global scope and it allows loading the specified module using its path relative to project folder, e.g `require('./src/components/graph/graphModel.js')`. **Note** that `require()` function is available in global scope only in debug build and not in production.
```
Comment (cornelissenja) [inline WebClient/doc/debugging.md]: typo? "accessible" should be "accessed"?

Comment (cornelissenja) [reply WebClient/doc/debugging.md]: or maybe: "can't be accessible directly" should be "are not directly accessible"?

File: WebClient/webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    return {
+        entry: './src/app.js',
+        output: {
+            filename: 'assets/js/[chunkhash].[name].bundle.js',
+            path: path.resolve(__dirname, 'dist')
+        },
+        module: {
+            rules: [
+                // html-loader
+                { test: /\.html$/, use: ['html-loader'] },
```
Comment (cornelissenja) [inline WebClient/webpack.config.js]: expand over multiple lines like below

File: WebClient/webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+                // css-loader
+                { 
+                    test: /\.css$/, 
+                    use: [
+                        MiniCssExtractPlugin.loader,
+                        {
+                            loader: 'css-loader',
+                            options: {
+                            sourceMap: devMode,
```
Comment (cornelissenja) [inline WebClient/webpack.config.js]: indentation incorrect

File: WebClient/webpack.config.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+                },
+                // file-loader
+                {
+                    test: /\.(jpg|png|gif|svg)$/,
+                    use: [
+                        {
+                            loader: 'file-loader',
+                            options: {
+                                name: '[name].[ext]',
+                                outputPath: './assets/media/'
```
Comment (cornelissenja) [inline WebClient/webpack.config.js:48]: Reminder: could be necessary to use a relative path or a definable base path

**Review by cornelissenja — APPROVED** [review]
ok with some small comments

File: WebClient/webpack.config.js (hunk 4 above)
Comment (korenkovv) [reply WebClient/webpack.config.js:48]: In this pull request the path is relative. I've also changed the config file in the 'details' branch and made math relative there as well.

**Review by rajatSinha303 — APPROVED** [review]
Everything looks perfect! 👍

**Pull request merged**